courses/dl2/imdb.ipynb

```diff
@@ -98,43 +150,190 @@
     "col_names = ['labels','text']"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use a random permutation np array to shuffle the text reviews."
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 125,
+   "execution_count": 15,
    "metadata": {},
    "outputs": [],
    "source": [
-    "np.random.seed(42)\n",
-    "trn_idx = np.random.permutation(len(trn_texts))\n",
-    "val_idx = np.random.permutation(len(val_texts))"
+    "np.random.seed(42) # the random seed ensures that the next 2 random permutations result in the same order.\n",
```

Comment:
That's unclear, or possibly wrong. The next 2 permutations have *different* orders. But they'll be the same each time you run the notebook.